user_docs/en/changes.t2t

```diff
@@ -37,7 +37,7 @@
  - "form": "frm"
  - "region": "rgn"
  - Example: "navigation landmark" becomes "lmk nav"
-- Espeak-ng has been updated to release 1.49.1 (#7280).
+- Updated Espeak-ng to 1.49.1. (#7280).

```

Comment:
1. This should be written "eSpeak NG" as is the official title of the project.
2. Please remove trailing full stop at end of line.